src/CommunityToolkit/Xamarin.CommunityToolkit/Views/BadgeView/BadgeView.shared.cs

```diff
@@ -230,7 +231,7 @@ void UpdateLayout()
 			BadgeIndicatorBackground.BorderColor = BorderColor;
 			BadgeIndicatorBackground.HasShadow = HasShadow;
 
-			BadgeText.Text = Text;
+			BadgeText.Text = string.IsNullOrEmpty(Text) ? "0" : Text;
```

Comment:
I don't think it's the proper way.
I think we should just delete the check inside UpdateVisibilityAsync

Reply:
I have tried with https://github.com/xamarin/XamarinCommunityToolkit/blob/1860937eacde3ced52e32fffff08a7592e8754b1/src/CommunityToolkit/Xamarin.CommunityToolkit/Views/BadgeView/BadgeView.shared.cs#L325-329 commented out but it was not working.

Reply:
maybe let's set here Text as it was before, but maybe we will hide indicator if the Value is "0" or null or emtpy?